src/main/java/com/uid2/shared/model/ServiceLink.java

```diff
@@ -12,12 +17,20 @@ public class ServiceLink {
     @JsonProperty("link_id")
     private String linkId;
     private String name;
+    private Set<Role> roles;
```

Comment:
Does anything need to be done to make this forward-compatible? E.g. what happens if a future json contains a role this binary does not yet know about?

Reply:
I've enabled the 'DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE' for ObjectMapper in ServiceLinkParser again as that allows the json to parse unknown roles as Role.UNKNOWN.